src/smart-components/workspaces/managed-selector/WorkspaceTreeView.tsx

```diff
@@ -0,0 +1,38 @@
+import { Spinner } from '@patternfly/react-core/dist/dynamic/components/Spinner';
+import { TreeView, TreeViewDataItem } from '@patternfly/react-core/dist/dynamic/components/TreeView';
+import * as React from 'react';
+import { TreeViewWorkspaceItem } from './TreeViewWorkspaceItem';
+
+interface WorkspaceTreeViewProps {
+  treeElements: TreeViewWorkspaceItem[];
+  areElementsFiltered: boolean;
+  selectedWorkspace?: TreeViewWorkspaceItem;
+  onSelect: (event: React.MouseEvent, item: TreeViewDataItem, parentItem: TreeViewDataItem) => void;
+  isLoading: boolean;
+}
+
+const WorkspaceTreeView = ({ treeElements, areElementsFiltered, selectedWorkspace, onSelect, isLoading }: WorkspaceTreeViewProps) => {
+  if (isLoading) {
+    return (
+      <div className="pf-v6-u-display-flex pf-v6-u-align-content-center">
```

Comment:
We usually wrap the spinner in the `Bullseye` PF component. Any reason why it would not work here?